Dockerfile.template

```diff
@@ -292,7 +292,9 @@ ENV GEM_HOME /usr/local/bundle
 ENV BUNDLE_SILENCE_ROOT_WARNING=1 \
 	BUNDLE_APP_CONFIG="$GEM_HOME"
 ENV PATH $GEM_HOME/bin:$PATH
-# adjust permissions of a few directories for running "gem install" as an arbitrary user
-RUN mkdir -p "$GEM_HOME" && chmod 1777 "$GEM_HOME"
+RUN set -eux; \
+	mkdir "$GEM_HOME"; \
```

Comment:
Do we expect all parent directories of `$GEM_HOME` to exist? Removing the `-p` causes `mkdir` to fail if the directory already exists, which is desireable, but it also means parent directories are not automatically created, which would be bad if all of `$GEM_HOME`'s parent directory do not exist in the image layers prior to this `RUN` command.

Reply:
It looks like all the Dockerfiles have a `mkdir -p /usr/local/etc` in a previous `RUN` statement, so this should be good.

Reply:
Indeed, and Ruby itself ends up in `/usr/local` via `make install`